pnpm-lock.yaml

```diff
@@ -92,7 +92,7 @@ importers:
       '@nuxt/module-builder': latest
       '@nuxt/test-utils': ^3.0.0-rc.8
       nuxt: ^3.0.0-rc.8
-      pinia: workspace:^2.0.18
+      pinia: '>=2.0.19'
```

Comment:
this needs to be reverted

Reply:
getting this error in build checks without this change(didn't manually change this. pnpm install did it)

 ERR_PNPM_OUTDATED_LOCKFILE  Cannot install with "frozen-lockfile" because pnpm-lock.yaml is not up to date with packages/nuxt/package.json

is it fine?

Reply:
No worries, I pushed a fix to the main branch